webknossos-datastore/app/com/scalableminds/webknossos/datastore/datareaders/jzarr/ZarrHeader.scala

```diff
@@ -1,64 +1,56 @@
-package com.scalableminds.webknossos.datastore.jzarr
-
-import com.scalableminds.util.geometry.{BoundingBox, Vec3Int}
+package com.scalableminds.webknossos.datastore.datareaders.jzarr
+
+import com.scalableminds.util.geometry.Vec3Int
+import com.scalableminds.webknossos.datastore.datareaders.{
+  ArrayDataType,
+  ArrayOrder,
+  BoolCompressionOption,
+  CompressionOption,
+  Compressor,
+  DatasetHeader,
+  DimensionSeparator,
+  IntCompressionOption,
+  StringCompressionOption
+}
 
 import java.nio.ByteOrder
-import com.scalableminds.webknossos.datastore.jzarr.ArrayOrder.ArrayOrder
-import com.scalableminds.webknossos.datastore.jzarr.BytesConverter.bytesPerElementFor
-import com.scalableminds.webknossos.datastore.jzarr.DimensionSeparator.DimensionSeparator
-import com.scalableminds.webknossos.datastore.jzarr.ZarrDataType.ZarrDataType
+import com.scalableminds.webknossos.datastore.datareaders.ArrayOrder.ArrayOrder
+import com.scalableminds.webknossos.datastore.datareaders.DimensionSeparator.DimensionSeparator
+import com.scalableminds.webknossos.datastore.datareaders.ArrayDataType.ArrayDataType
 import com.scalableminds.webknossos.datastore.models.datasource.{DataLayer, ElementClass}
-import net.liftweb.common.Box.tryo
+import net.liftweb.util.Helpers.tryo
```

Comment:
This changed because my IDE can't find `tryo` in `net.liftweb.common.Box`. Since I also can't any implementation there, I am not sure what's different...